src/tools/rust-analyzer/bench_data/glorious_old_parser

```diff
@@ -1826,7 +1826,7 @@ impl<'a> Parser<'a> {
             "expected a path on the left-hand side of `+`, not `{}`", pprust::ty_to_string(ty));
 
         match ty.node {
-            TyKind::Rptr(ref lifetime, ref mut_ty) => {
+            TyKind::Ref(ref lifetime, ref mut_ty) => {
```

Comment:
Can we keep this file unchanged? It's a rust-analyzer benchmark and doesn't need to be continuously updated.

Reply:
sounds like a good idea